### PR TITLE
Transit Import Key CLI functionality

### PR DESCRIPTION
**Pull request by sgmiller**
Adds transit import/import-key to bring keys into transit and transform via
the key wrapping protocol.

### CODE REVIEW

File: command/transit_import_key.go [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	c.UI.Output("Wrapping source key with ephemeral key.")
+	wrapKWP, err := subtle.NewKWP(ephemeralAESKey)
+	if err != nil {
+		c.UI.Error(fmt.Sprintf("failure building key wrapping key: %v", err))
+		return 2
+	}
+	wrappedTargetKey, err := wrapKWP.Wrap(key)
+	if err != nil {
+		c.UI.Error(fmt.Sprintf("failure wrapping source key: %v", err))
+		return 2
```
Comment (kitography) [inline command/transit_import_key.go:116]: what are your return codes / are they documented somewhere? (in particular 1 vs. 2 vs 3 as failures)

**Review by kitography — COMMENTED** [review]
Looks pretty good!

Are there tests here anywhere?

**Comment by sgmiller** [comment]
Apparently cli commands don't have unit tests?  The actual import algo is covered by the unit tests in transit though.

File: command/transit_import_key.go (hunk 1 above)
Comment (sgmiller) [reply command/transit_import_key.go:116]: Will do.

File: command/transit_import_key.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+		return nil, fmt.Errorf("transit not mounted at %s: %v", path)
+	}
+	key, ok := resp.Data["public_key"]
+	if !ok {
+		c.UI.Error("could not find wrapping key")
+	}
+	keyBlock, _ := pem.Decode([]byte(key.(string)))
+	parsedKey, err := x509.ParsePKIXPublicKey(keyBlock.Bytes)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing wrapping key: %v", err)
```
Comment (stevendpclark) [inline command/transit_import_key.go:169]: nit: 
```suggestion
		return nil, fmt.Errorf("error parsing wrapping key: %w", err)
```

**Review by stevendpclark — APPROVED** [review]

**Comment by cipherboy** [comment]
> Apparently cli commands don't have unit tests?

CLI does have unit tests, see e.g., `pki_health_check_test.go` or `pki_verify_sign_test.go` for more complicated ones. :-)

**Pull request merged**